src/pudl/metadata/codes.py

```diff
@@ -2295,6 +2295,133 @@
         "code_fixes": {},
         "ignored_codes": [],
     },
+    "core_eia__codes_cooling_tower_types": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    "MD",
+                    "mechanical_dry",
+                    "Mechanical draft, dry process",
```

Comment:
The docs build is failing because I need to add these to the `RESOURCE_METADATA` table in `metadata/resources/eia.py`. 

I added it, but it's causing dagster to fail, I'm not exactly sure why.